### PR TITLE
Feat update to hamlet

### PR DESCRIPTION
**Pull request by RossMurr4y**
## Description

Remove references to `codeontap`.

I have updated a number of references to the purpose of the file, but this was only so the words flow - this is not intended to be a revision of the documents in their entirety.

## Motivation and Context

Removing remnants of codeontap references from Hamlet.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] Refactor (non-breaking change which improves the structure or operation of the implementation)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Followup Actions

- [x] None

## Checklist:

- [ ] My change requires a change to the [documentation](https://github.com/hamlet-io/docs).
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [x] None of the above.

### CODE REVIEW

**Comment by RossMurr4y** [comment]
Looks like this repo doesn't have the usual groups on it. @ml019 would you mind reviewing?

File: products/app-lifecycle-mgmt/{{cookiecutter.product_name}}/config/settings/{{cookiecutter.environment_name}}/{{cookiecutter.segment_name}}/jenkins/asFile/site.properties [hunk 1]
```diff
@@ -1,8 +1,8 @@
-AUTOMATION_BASE_DIR=/opt/codeontap/automation
-AUTOMATION_DIR=/opt/codeontap/automation/jenkins/aws
-GENERATION_BASE_DIR=/opt/codeontap/generation
-GENERATION_STARTUP_DIR=/opt/codeontap/startup
-GENERATION_DIR=/opt/codeontap/generation/aws
+AUTOMATION_BASE_DIR=/opt/hamlet/automation
+AUTOMATION_DIR=/opt/hamlet/automation/jenkins/aws
+GENERATION_BASE_DIR=/opt/hamlet/generation
+GENERATION_STARTUP_DIR=/opt/hamlet/startup
+GENERATION_DIR=/opt/hamlet/generation/aws
```
Comment (roleyfoley) [inline products/app-lifecycle-mgmt/{{cookiecutter.product_name}}/config/settings/{{cookiecutter.environment_name}}/{{cookiecutter.segment_name}}/jenkins/asFile/site.properties]: These should probably be updated to align with what we have in the hamlet container I don't think they are quite right there

File: products/app-lifecycle-mgmt/{{cookiecutter.product_name}}/config/solutionsv2/shared/{{cookiecutter.segment_name}}/solution.json [hunk 2]
```diff
@@ -76,10 +76,10 @@
                                             }
                                         },
                                         "Links" : {
-                                            "slave-codeontap" : {
+                                            "slave-hamlet" : {
```
Comment (roleyfoley) [inline products/app-lifecycle-mgmt/{{cookiecutter.product_name}}/config/solutionsv2/shared/{{cookiecutter.segment_name}}/solution.json]: We should also rename slave to agent

**Review by roleyfoley — CHANGES_REQUESTED** [review]

**Comment by roleyfoley** [comment]
@RossMurr4y Group access permissions have now been fixed up as well so you should be able to assign @hamlet-io/engine-maintainers

File: products/app-lifecycle-mgmt/{{cookiecutter.product_name}}/config/settings/{{cookiecutter.environment_name}}/{{cookiecutter.segment_name}}/jenkins/asFile/site.properties (hunk 1 above)
Comment (RossMurr4y) [reply products/app-lifecycle-mgmt/{{cookiecutter.product_name}}/config/settings/{{cookiecutter.environment_name}}/{{cookiecutter.segment_name}}/jenkins/asFile/site.properties]: updated here: https://github.com/hamlet-io/executor-cookiecutter/pull/8/commits/7befc4b32aef36c535ae9cfaed1a4798038d3d00

File: products/app-lifecycle-mgmt/{{cookiecutter.product_name}}/config/solutionsv2/shared/{{cookiecutter.segment_name}}/solution.json (hunk 2 above)
Comment (RossMurr4y) [reply products/app-lifecycle-mgmt/{{cookiecutter.product_name}}/config/solutionsv2/shared/{{cookiecutter.segment_name}}/solution.json]: Doneski https://github.com/hamlet-io/executor-cookiecutter/pull/8/commits/471f02441880af09fa5c2b8588f1cd63b112831f

**Review by roleyfoley — APPROVED** [review]

**Pull request merged**